db/migrate/20190716173959_remove_items.rb

```diff
@@ -0,0 +1,15 @@
+class RemoveItems < ActiveRecord::Migration[5.2]
+  def change
```

Comment:
Shouldn't we migrate existing vocab sheets here too?

Reply:
I assumed we wouldn't bother with data migration given that vocab sheets are quite short lived but maybe that's being a bit 🤠 ?

Reply:
i guess we don't know. We could do some analysis on the data to work out the usage patterns of people who use vocab sheets